### ISSUE TITLE
Android SDK - no onAdFailed() action for paused ad units

### ISSUE THREAD

**Issue by caynax** [AppLovin/AppLovin-MAX-SDK-Android#540]
### MAX SDK Version

11.11.3

### Device/Platform Info

All

### Current Behavior

Hi.
Some time ago I have turned off all banner units for your network.
Unfortunately it turns out that in such case no onAdFailed() is called - even in latest 11.11.3 Android SDK.
Simply nothing happens and app hangs with none ad loaded as you didn't pass on adFailed() information.

I see that in logs:

NO FILL received:
..ID: "962f1c81da13afd3"
..FORMAT: "BANNER"
..SDK KEY: "xxxxxxx"
..PACKAGE NAME: "com.caynax.sportstracker"
..Reason: [{"error_source":"INV-REQ-INELIGIBLE-ADUNIT-ID","code":1035,"msg":"Invalid Request: Ineligible Ad Unit"}]

but no onAdFailed() is called so application doesn't know that.

### Expected Behavior

Call onAdFailedToLoad().

### How to Reproduce

Call banner ad with ID that is paused.

**Comment by thomasmso** [comment]
@caynax - If you wait 1-2 minutes, the callback will eventually get fired. The SDK is attempting retries internally, hence the delayed callback. Let me know if that is not what you are observing.

**Comment by caynax** [comment]
That's too long. App is left without ads = not monetized. I got you mixed
with AdMob and I was wondering why some users write that "this is a great
app without advertisements". It was fine while app was loading AdMob ad.
While it tried to AppLovin ad it got stuck without ads. Luckily I got all
ad networks configured via own online system so I simply removed AppLovin
from that list (which I forgot to do after pausing all banner ads) but I
would expect adFailedToLoad() fired immediately so that I could try another
ad (from other ad network like AdMob).
